app/presenters/route_summary_card_data_presenter.rb

```diff
@@ -79,10 +82,27 @@ def default_route_card(index)
     }
   end
 
+  def edit_secondary_skip_link
+    if Settings.features.branch_routing && all_routes.find(&:secondary_skip?).present?
+      govuk_link_to("Edit", edit_secondary_skip_path(form_id: form.id, page_id: page.id))
+    end
+  end
+
   def secondary_skip_rows
     secondary_skip = all_routes.find(&:secondary_skip?)
 
-    return [] if secondary_skip.blank?
+    if secondary_skip.blank?
+      if Settings.features.branch_routing
```

Comment:
I think we prefer to use the `FeatureService` to check if a feature is enabled:

```suggestion
      if FeatureService.enabled?(:branch_routing)
```

Reply:
Good point, I think there's another spot I can refactor that as well